Dockerfile

```diff
@@ -6,23 +6,37 @@ COPY package.json ./
 RUN npm i
 
 # bundle install the gems for production
-FROM ruby:3.3.5 as rubybuilder
-RUN apt update -y \
-    && apt -y install nano \
-    && cp /usr/bin/nano /usr/local/bin/
+FROM ruby:3.3.5-alpine as rubybuilder
```

Comment:
Is the switch to `-alpine` intentional here? 

[alpine linux](https://www.alpinelinux.org/about/) is a lightweight linux distribution designed for running in containers. It uses a different 'libc' than debian (which is what the `-slim`) version uses. 

There may be compatibility issues arising from pulling in from a different linux distribution. 

It would probably be safest stick with the `-slim` variant for any additional builder layers that are pulled in

Reply:
We're looking at a switch to alpine as the `ruby:3.3.5` and `ruby:3.3.5-slim` images are based on debian stable, which appears to have a critical CVE associated with it. It looks like Hauwa has been able to get the app running correctly with alpine and the `gcompat` library, which handles the glibc/musl conversion.

I think this is a really good change if it works as it gives us a much smaller image and a more up-to-date set of packages, with a smaller attack surface and therefore fewer vulnerabilities.